A/alsa_plugins/build_tarballs.jl

```diff
@@ -3,32 +3,31 @@
 using BinaryBuilder, Pkg
 
 name = "alsa_plugins"
-version = v"1.2.2"
+version = v"1.2.7"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("ftp://ftp.alsa-project.org/pub/plugins/alsa-plugins-1.2.2.tar.bz2", "1c0f06450c928d711719686c9dbece2d480184f36fab11b8f0534cb7b41e337d")
+    ArchiveSource("https://www.alsa-project.org/files/pub/plugins/alsa-plugins-1.2.7.1.tar.bz2",
+                  "8c337814954bb7c167456733a6046142a2931f12eccba3ec2a4ae618a3432511"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir
-cd alsa-plugins-*
+cd $WORKSPACE/srcdir/alsa-plugins-*
 ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
 make
 make install
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = filter!(Sys.islinux, supported_platforms())
-
+platforms = filter(p -> Sys.islinux(p) & (arch(p) != "armv6l"), supported_platforms(;experimental=true))
 
 # The products that we will ensure are always built
 products = [
-    LibraryProduct("libasound_module_pcm_a52", :libasound_module_pcm_a52, "lib/alsa-lib"),
+    LibraryProduct("libasound_module_pcm_a52", :libasound_module_pcm_a52, "lib/alsa-lib"; dont_dlopen=true),
```

Comment:
Why don't dlopen?

Reply:
Doesn't sound very useful if we can't dlopen it?  Why it used to work?  What's the problem exactly?